udata/templates/search.html

```diff
@@ -134,7 +138,15 @@
 {% block main_content %}
 <div class="tab-content">
 
-    <div class="tab-pane row active" id="datasets">
+    {% if not datasets and not reuses and not organizations and not users and not territories %}
```

Comment:
When not any results, the tabs are still displayed. Doesn't it break the layout ?